frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.tsx

```diff
@@ -138,6 +138,18 @@ export const ParametersWidgetContainer = styled(FullWidthContainer)<{
     `}
 `;
 
+export const FIXED_WIDTH = "1048px";
```

Comment:
Not sure if it's expected or not, but even though we're setting 1048px, the actual content is a bit narrower because of some horizontal margin. I think we'd want to remove it @kdoh @cdeweyx ? I want these 28px back! 😂 

![CleanShot 2024-02-02 at 18 09 50@2x](https://github.com/metabase/metabase/assets/17258145/a80e3f61-3d3b-4847-9d18-154eb138b1c5)

Reply:
After I moved `FixedWidthContainer` into `DashboardGridConnected`, this is no longer an issue. It still has padding when it reaches the edges of the viewport, but that is where we want it.